### ISSUE TITLE
Up/Download on new keyserver do not work

### ISSUE THREAD

**Issue by webratte** [open-keychain/open-keychain#2469]
## Expected Behavior

It should be possible Upload own public key and also download of other keys

## Current Behavior
Error message -- see attached screenshot

## Steps to Reproduce (for bugs)

1. Try to upload your own key

## Your Environment

* Android Version: 7.0
* Device Model: Huawei nova
* OpenKeychain Version: 5.3
* From Google Play or F-Droid?: Google play
![Openkeychain](https://user-images.githubusercontent.com/6864851/60532273-3ae11800-9cfd-11e9-844f-ab2dad833899.jpg)

**Comment by Valodim** [comment]
This works just fine for me. You mention that download also doesn't work?

Just making sure, does it work to go to keys.openpgp.org in a browser on your phone?

**Comment by webratte** [comment]
Just tried it on the same device in Mobil Firefox.
I can upload my key and find it also.

So in the browser it works fine.

Is there a easy way to create a logfile where you can find out what happens?

**Comment by Valodim** [comment]
no, sorry. perhaps it's something about your key? can you send it to me please? -> `look@my.amazin.horse`

**Comment by webratte** [comment]
Hi,
I sendet me keys to you.

And just I tried to find your key vie OpenKeychain and it failed.

Via Browser I find it.

So it seems not a problem with the keys.

**Comment by dschuermann** [comment]
I just had the idea that this could be caused when the TLS connection to the server fails. But since Android 5, TLS 1.2 should work: https://developer.android.com/about/versions/android-5.0-changes.html#ssl

Still, maybe there is something failing at TLS, maybe the Lets Encrypt root cert is missing in Android's trust store on your device.

We should test these cases.

**Comment by Valodim** [comment]
I indeed made a change wrt TLS recently that added more compatible ciphersuites. Can you check if the error persists?

**Comment by polyzen** [comment]
Perhaps related to https://keys.openpgp.org/about/faq#older-gnupg ?

**Comment by Valodim** [comment]
Good thought, but in that case it wouldn't be a network error. Must be something else

**Comment by radasbona** [comment]
Something is indeed wrong.

In the app i uploaded my key to the new server, without any errors (i deleted all the other servers). But on the Website Search my key is not found.

**Comment by webratte** [comment]
Hi, 
have you some news about this?
I have just tried to deinstall and reinstall OpenKeychain.
But still it dosn't work.

**Comment by 51ckb01** [comment]
I think I stumbled into this situation today:

1. I added the server correctly
2. I was able to upload without errors
3. I tried to look for my key on the site, but couldn't find anything.

I wanted to do a manual upload test and I noticed that, in order to be visible by email, the site asks to send an identity confirmation email.

Once the identity is confirmed, my key is searchable via email address.

I noticed that on the site, at the following [Link](https://keys.openpgp.org/about/usage) says:

> Note that there is no support prepared for loading and verification of email addresses for now.
